File: pset1/mario/mario.c
#include<stdio.h>
#include<cs50.h>

int main(void)
{
    int height;
    while (1)
    {
        height = get_int("Height: ");
        if (height >= 1 && height <= 8)
        {
            break;
        }
    }
    for (int i = 1; i <= height; i++)
    {
        for (int j = height; j > i; j--)
        {
            printf(" ");
        }
        for (int j = 1; j <= i; j++)
        {
            printf("#");
        }
        printf("  ");
        for (int j = 1; j <= i; j++)
        {
            printf("#");
        }
        printf("\n");
    }
}

File: pset6/mario/more/mario.py
import cs50

while True:
    height = cs50.get_int("Height: ")
    if height > 0 and height <= 8:
        break

for i in range(1, height + 1, 1):
    print(" " * (height - i) + "#" * i + "  " + "#" * i)